### PR TITLE
DigitalGBPToken and successful tests

### PR DESCRIPTION
**Pull request by DhenPadilla**
- Added a separate tests file for the DigitalGBPToken.

- 5 Tests, All pass

- 1 test checking whether an error is thrown when burning too many tokens.

### CODE REVIEW

File: test/digital_g_b_p_token.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
+const DigitalGBPToken = artifacts.require("DigitalGBPToken");
+
+contract('DigitalGBPToken', accounts => {
+
+  var token;
+  var _name = 'DigitalGBPToken';
+  var _symbol = 'DGBP';
+  var _dec = 2;
+
+  const ERROR_MSG = 'VM Exception while processing transaction: revert';
```
Comment (DhenPadilla) [inline test/digital_g_b_p_token.js]: Removed this, also:

`const ERROR_MSG = 'VM Exception while processing transaction: revert';`

Comment (jakub-wojciechowski) [reply test/digital_g_b_p_token.js]: You can use .shouldBeReverted assertion

File: test/digital_g_b_p_token.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    });
+    it('Should Mint 100, then Burn 100', async function() {
+      token.mint(accounts[1], 100).then(async function() {
+        await token.burn(accounts[1], 100);
+        const res = await token.balanceOf(accounts[1]);
+        console.log("error: " + res + "\n");
+        assert.strictEqual(res, 0, 'Burnt 100');
+      });
+    });
+    it('Reject Burn when burning more than total supply', async function() {
```
Comment (jakub-wojciechowski) [inline test/digital_g_b_p_token.js]: You can also use the .shouldBeReverted assertion to check if an error has been throwed

Comment (jakub-wojciechowski) [reply test/digital_g_b_p_token.js]: Please make a space (empty line) between functions. It easier to read that way (and looks nicer too ;))

File: test/digital_g_b_p_token.js [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+  before('deploy DigitalGBPToken', async function() {
+    token = await DigitalGBPToken.deployed();
+  });
+
+
+  describe('Token Attributes', function() {
+    it('Has correct name', async function() {
+      const name = await token.name();
+      assert.equal(name, _name, 'Name is correct');
```
Comment (jakub-wojciechowski) [inline test/digital_g_b_p_token.js]: We tend to use chai assertions in all of the other tests, imho is more readable.

File: test/digital_g_b_p_token.js (hunk 2 above)
Comment (jakub-wojciechowski) [inline test/digital_g_b_p_token.js]: I'll also test a more narrow constraint with burning more than the account balance

Comment (jakub-wojciechowski) [inline test/digital_g_b_p_token.js]: It'll be also good to test if minting a burning could be called only by the owner.

**Review by jakub-wojciechowski — CHANGES_REQUESTED** [review]
Great job, please take a look at the comments.

**Pull request merged**